modules/create-a-containerruntimeconfig-crd.adoc

```diff
@@ -15,7 +15,7 @@ To revert the changes implemented by using a `ContainerRuntimeConfig` CR, you mu
 
 You can modify the following settings by using a `ContainerRuntimeConfig` CR:
 
-* **PIDs limit**: Setting the PIDs limit in the `ContainerRuntimeConfig` is expected to be deprecated. If PIDs limits are required, it is recommended to use the `podPidsLimit` field in the `KubeletConfig` CR instead. The default value of the `podPidsLimit` field is `4096`.
+* **PIDs limit**: Setting the PIDs limit in the `ContainerRuntimeConfig` is deprecated. If PIDs limits are required, it is recommended to use the `podPidsLimit` field in the `KubeletConfig` CR instead. The default value of the `podPidsLimit` field is `4096`.
```

Comment:
```suggestion
* **PIDs limit**: Setting the PIDs limit in the `ContainerRuntimeConfig` CR is deprecated. If PIDs limits are required, use the `podPidsLimit` field in the `KubeletConfig` CR instead. The default value of the `podPidsLimit` field is `4096`.
```
- Per IBM Style, the term "recommend" is a use-with-caution word and should be avoided if possible. See https://www.ibm.com/docs/en/ibm-style?topic=word-usage#r.